### ISSUE TITLE
Back-fill `hashid` for voters where it's missing

### ISSUE THREAD

**Issue by sjforman** [sjforman/votefwd#97]
Evidence of the problem as of 10/17: 

```
votefwd=> select count(*) from voters where hashid is null and adopted_at is not null;
 count 
-------
 30685
(1 row)
```

The `hashid` for these voters *was* rendered onto the generated PDFs, which is good (see comment below for Jeff's explanation of why that is so). 

TODO:
 
 * (if easy) ensure that `hashid`s are properly persisted?
 * Back-fill the missing ones. 

Notes: 

 * Might be best to insert the `hashid` when each voter is added to the `voters` table in the first place, but this currently happens as a quasi-manual process using postgres stored procedures, so this is probably not straightforward in the short term.
 * For now, the back-filling could just be a manual process to run (carefully) against the database one time on October 30. If we can do this, we might not even need to worry, for now, about making sure they're always persisted.
 * I took a small stab in this direction in https://github.com/sjforman/votefwd/tree/backfill-missing-hashids but haven't finished it yet.

**Comment by jeffkole** [comment]
The `storeHashIdForVoter` function is a "fire and forget" function, meaning, it logs an error if one occurs, but it does not affect the control flow of the logic. Since the hashid is generated before that function is called, it will always get put on the PDF even if the db update fails. Fortunately, we should be able to find errors associated with the function call in the logs.

**Comment by carlmanaster** [comment]
Looks to me like we could write an endpoint which:

- gets the list of hashid-less voters
- iterates over them, calling `storeHashIdForVoter`

Would this approach work?

**Comment by sjforman** [comment]
Yes, I think so! That seems like a wiser approach than what I started to do, which was to pull them out of the database, generate the hashIds, and then re-insert them. Could you take this one @carlmanaster?

**Comment by carlmanaster** [comment]
Sure!

**Comment by carlmanaster** [comment]
https://github.com/sjforman/votefwd/tree/fix-voter-hashes

might do it.  I have been unable to test locally, so I have little confidence in this code, but have no more time to work on it this afternoon; please feel free to give it a try.  I'll try to get back to it this evening.

**Comment by jeffkole** [comment]
You can write a knex migration script to do this too. But if we expect hashids to be missing regularly then an endpoint to fix em sounds good

Jeff

**Comment by sjforman** [comment]
Closed by #265
